README.md

```diff
@@ -2,12 +2,15 @@
  ________________________________________
 ( This repo contains info about projects )
 ( and repositories tracked by the        )
-( Bitergia Analytics Dashboard. For more )
-( info visit http://bitergia.com         )
+( Bitergia Analytics Dashboard.          )
  ----------------------------------------
         o   ^__^
          o  (oo)\_______
             (__)\       )\/\
                 ||----w |
                 ||     ||
 ```
+# How to Contribute
+* The list is generated from this spreadsheet: https://docs.google.com/spreadsheets/d/1xFrt9JZkZ6m8peasYaRfPO5kC1JDo-vDJJuYfMwcGqM/edit
+* Please do not submit PRs here but instead edit/comment the spreadsheet. 
+* Ping us on the Community Analytics Telegram channel to trigger a refresh of the list
```

Comment:
Should we post an invite link for that group here? If not, can you add a description on how to get invited to that group?